.github/workflows/copybara.yaml

```diff
@@ -18,23 +18,28 @@ on:
   push:
     branches:
     - copybara
+
 jobs:
   create_pr:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
-    - name: Checkout a pull request from copybara to main
+    - name: Check out repo
+      uses: actions/checkout@v3
+      with:
+        ref: 'copybara'
+    - name: Create pull request for changes
+      uses: googleapis/code-suggester@v2
       env:
-        GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
-      run: |
-        # Only create a PR if DNE
-        if ! [[ `gh pr list --base main --search "[Copybara]"` ]]; then
-          git config --global user.email "yoshi-code-bot@google.com"
-          git config --global user.name "Yoshi Code Bot"
-          gh pr create \
-          --base main \
-          --head copybara \
-          --title "[Copybara] Update with new internal changes from Piper" \
-          --body "This is a auto-generated pull request. The PR updates the default branch with new changes from the copybara branch. Refer to PR commit history for more details." \
-          --label "cla: yes"
-        fi
\ No newline at end of file
+        ACCESS_TOKEN: ${{ secrets.YOSHI_CODE_BOT_TOKEN }} # manually created and added as repo secret
+      with:
+        command: pr
+        upstream_owner: ${{ github.repository_owner }}
+        upstream_repo: 'google-cloudevents'
+        description: 'Generated in GitHub action: https://github.com/${{ github.repository }}/actions/workflows/copybara.yaml'
+        title: 'feat(copybara): Update with new internal changes from Piper'
+        message: 'feat(copybara): Update with new internal changes from Piper'
+        branch: 'copybara-${{ github.run_id }}' # branch for forked repo, don't overwrite changes
```

Comment:
I'm curious how these fields work? Is there some Github guide I can use to figure out what these commands do? I'm trying to use https://cli.github.com/manual/gh_pr, but it seems like the commands are different here.

Reply:
This action is built and recommended by the DPE team, see https://github.com/googleapis/code-suggester#Action for config. The bot account will fork the repo with changes on the `copybara-NNN` branch so changes won't be overwritten. Though, it looks like I should remove the git_dir and force fields (copy/paste errors).